bpf/cpu/cpu.bpf.c

```diff
@@ -174,9 +174,10 @@ u32 UNWIND_SAMPLES_COUNT = 7;
 BPF_HASH(debug_pids, int, u8, 32);
 BPF_HASH(stack_counts, stack_count_key_t, u64, MAX_STACK_COUNTS_ENTRIES);
 BPF_STACK_TRACE(stack_traces, MAX_STACK_TRACES);
-BPF_HASH(dwarf_stack_traces, int, stack_trace_t, MAX_STACK_TRACES);
+BPF_HASH(dwarf_stack_traces, int, stack_trace_t,
+         2); // Table size will be updated in userspace.
```

Comment:
I think we might want to keep this number as-is, for the same reason why we want to have some items in the `unwind_tables` map. Otherwise, we might be locking enough memory to be representative enough for the future release.

I realise that this is a waste of memory, but I believe it will help us ensure we test as many things ahead of turning DWARF unwinding by default.

As discussed, in the future, we will probably have just one map for `hash->stack` no matter the used method to unwind the stack

Reply:
Just to make it clearer, given that in the next release DWARF unwinding will be enabled by default and both `dwarf_stack_traces` and `stack_traces` will be allocated and used, I think it's better to start allocating these maps as soon as possible so we can fix any issues that might arise 😄

Reply:
👍 The gain is negligible anyway.